apps/website/docusaurus.config.js

```diff
@@ -1,6 +1,13 @@
 // @ts-check
 
-const darkCodeTheme = require('prism-react-renderer/themes/vsDark');
+const vsDarkTheme = require('prism-react-renderer/themes/vsDark');
+const prismTheme = {
+  ...vsDarkTheme,
+  plain: {
+    ...vsDarkTheme.plain,
+    backgroundColor: '#000',
+  },
+};
```

Comment:
This will be a fully custom theme. Currently tweaked to apply proper background.